### PR TITLE
LPS-166804 Change user card background color when mark as answer

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for the [LPS-166804](https://issues.liferay.com/browse/LPS-166804). Please help me to review and give feedback.

Thanks!

### CODE REVIEW

**Comment by kevenleone** [comment]
Just starting reviewing :)

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -74,6 +74,9 @@
 		background-color: $success-l2;
 		border: 1px solid $success-l1;
 		border-radius: 0.25rem;
+		& .btn-secondary {
+			background-color: unset;
+		}
 	}
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss:80]: Just need to add spaces

```suggestion
		border-radius: 0.25rem;
		
		&.btn-secondary {
			background-color: unset;
		}
	}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@sontruongces thank you for submitting this PR, It's good!

I'd like to suggest only spaces

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (sontruongces) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss]: Thank @kevenleone, I updated the code.

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
Thanks @sontruongces 

Forwarding to Bchan

**Pull request closed without merging**